### PR TITLE
0x06. Unittests in JS 

### PR DESCRIPTION
**Pull request by the1Riddle**
`UnitTests` `Back-end` `JavaScript` `ES6` `NodeJS` `ExpressJS` `Mocha`

### CODE REVIEW

File: 0x06-unittests_in_js/0-calcul.js
Comment (the1Riddle) [inline 0x06-unittests_in_js/0-calcul.js:1]: Basic test with Mocha and Node assertion library

File: 0x06-unittests_in_js/1-calcul.js
Comment (the1Riddle) [inline 0x06-unittests_in_js/1-calcul.js:1]: Combining descriptions

File: 0x06-unittests_in_js/10-api/.gitignore
Comment (the1Riddle) [inline 0x06-unittests_in_js/10-api/.gitignore:1]: git ignore did not ignore itself LoL

File: 0x06-unittests_in_js/10-api/api.js
Comment (the1Riddle) [inline 0x06-unittests_in_js/10-api/api.js:1]: Deep equality & Post integration testing

**Review by the1Riddle — COMMENTED** [review]
reviewing your own work is awesome

**Review by rezzcode — APPROVED** [review]
:+1:

**Pull request merged**